src/dmtcp_coordinator.cpp

```diff
@@ -120,6 +120,8 @@ static const char* theUsage =
   "  -i, --interval (environment variable DMTCP_CHECKPOINT_INTERVAL):\n"
   "      Time in seconds between automatic checkpoints\n"
   "      (default: 0, disabled)\n"
+  "  -q, --quiet \n"
+  "      Skip copyright notice.\n"
```

Comment:
We don't print the copyright notice at all, so let's remove this statement. We can replace it with something else that mentions the quietness.